src/target/llvm/llvm_module.cc

```diff
@@ -234,7 +238,7 @@ class LLVMModuleNode final : public runtime::ModuleNode {
       }
       funcs.push_back(f);
     }
-    ICHECK(funcs.size() > 0 || (could_have_linked_params && found_linked_params));
+    // ICHECK(funcs.size() > 0 || (could_have_linked_params && found_linked_params));
```

Comment:
This should still be a valid check, no?

Reply:
I think we are allowed to have an empty model now? I can restore but I remember this no longer being true